### DISCUSSION TITLE
是不是可以把烂尾楼或者停贷涉及的主要银行也统计一下？

### DISCUSSION THREAD

**Discussion by georgecao** [WeNeedHome/SummaryOfLoanSuspension#847] [Ideas]
如题，就像开发商一下，烂尾楼也会涉及开发商背后的主要银行，这部分数据是否也可以收^_^收集一下？

**Comment by leafgrey** [comment]
我也在想 也搜索了一下 好像已经有人做了 
不太统一
https://github.com/ritajie/incomplete-projects
https://github.com/taozhijiang/dust_repos
